backstop/src/backstop/user.rs

```diff
@@ -51,10 +51,12 @@ impl UserBalance {
         if self.shares < to_q {
             panic_with_error!(e, BackstopError::BalanceError);
         }
+        if self.q4w.len() >= MAX_Q4W_SIZE {
```

Comment:
y 10 max

Reply:
No particular reason - I kept it fairly restrictive as 10 means users are creating a new Q4W every 2 days before they can start withdrawing to create more (which seems excessive).

I can increase this if you feel it's too restrictive.

Resource wise, it could handle at least 50, but I'd need to perform some tests to ensure it's safe after that.

Reply:
We decided that increasing max_q4w_size to 21 (at most 1 entry a day for the 21 day lock period) felt more reasonable than this.

Commit was updated.